main.go

```diff
@@ -65,7 +66,19 @@ func init() {
 	//+kubebuilder:scaffold:scheme
 }
 
+type StringSliceVar []string
```

Comment:
Why is that not built in??

Reply:
There is something like this in the [`spf13/pflag`](https://pkg.go.dev/github.com/spf13/pflag#StringSliceVar) library, but this does not work, but this does not works with controller-runtime `BindFlags` function :shrug: